### PR TITLE
README & info.xml: Add ethical AI rating

### PR DESCRIPTION
**Pull request by marcelklehr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
-This app includes 2 custom link pickers for Text, Talk and potentially anywhere in Nextcloud:
+This app includes 3 custom smart pickers for Nextcloud:
 * ChatGPT answers
 * DALL·E 2 images
+* Whisper dictation
+
+It also implements
+
+* a Translation provider (using ChatGPT)
+* a SpeechToText provider (using Whisper)
```
Comment (julien-nc) [inline README.md:11]:
```suggestion
* A Translation provider (using ChatGPT)
* A SpeechToText provider (using Whisper)
```

Uppercase start for bullet points...?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,45 @@
...
+* the software for training and inferencing of this model is proprietary, limiting running it locally or training by yourself
+* the trained model is not freely available, so the model can not be ran on-premises
+* the training data is not freely available, limiting the ability of external parties to check and correct for bias or optimise the model’s performance and CO2 usage.
+
+
+### Rating for Whisper via OpenAI API: 🟡
+
+Positive:
+* the software for training and inferencing of this model is open source
+* the trained model is freely available, and thus can be ran on-premises
```
Comment (julien-nc) [inline README.md:36]:
```suggestion
* The trained model is freely available, and thus can run on-premise
```
Just a suggestion.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,45 @@
...
+
+Instead of connecting to the OpenAI API for these, you can also connect to a self-hosted LocalAI instance.
+
+## Ethical AI Rating
+### Rating for ChatGPT via OpenAI API: 🔴
+
+Negative:
+* the software for training and inference of this model is proprietary, limiting running it locally or training by yourself
+* the trained model is not freely available, so the model can not be run on-premises
+* the training data is not freely available, limiting the ability of external parties to check and correct for bias or optimise the model’s performance and CO2 usage.
```
Comment (julien-nc) [inline README.md:21]:
```suggestion
* the training data is not freely available, limiting the ability of external parties to check and correct for bias or optimise the model's performance and CO2 usage.
```
Nitpicking :grin: Any reason to not use the classic simple quote?

**Review by julien-nc — COMMENTED** [review]
Thanks a lot! :blue_heart: :+1: 

A few minor remarks.

**Review by DaphneMuller — APPROVED** [review]
thank you!

**Pull request merged**